README.md

```diff
@@ -105,8 +105,25 @@ Yamashita | Yah-MAH-sh-Ta  | some vowels (i, u) get removed (called devoicing),
 Kyoko     | Kyoh-koh       | (not Kee-oh-koh)      
 Ryousuke  | Ryoh-skeh      | the -suke suffix is usually skeh
 
+## Portuguese (Brazilian)
+
+The challenge with Portuguese is that there are some phonems (such as \[ɐ̃w̃\], \[s̻\] or \[ʎ\]) and [diacritics](https://en.wikipedia.org/wiki/Diacritic) (such as an accent or cedilla) not common or not at all present in the English language. 
+
+First, a few general rules:
+1) **ão** is a nasal diphthong and resembles "pawn" + "o"
+2) **ç** is like a soft S sound (think of how you'd prounounce the French word [facade](https://en.wiktionary.org/wiki/facade))
+3) **lh** is similar to the Spanish "ll" as in "pollo" (chicken) or "pol" - "io"
```

Comment:
can we lead with the easier "it's basically the y in yellow" and then add the detail that it's the same as ll in spanish as color?   i would like to lead with the easiest approximation and then follow with more correct answer.